backend-rust/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "concordium-scan"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1"
+async-graphql = {version = "7.0", features = ["chrono", "tracing"] }
+async-graphql-axum = "7.0"
+axum = "0.7"
+ciborium = "0.2"
+chrono = { version = "0.4", features = ["serde"] }
+clap = { version = "4.5", features = ["derive", "env"] }
+concordium-rust-sdk = { path = "./concordium-rust-sdk" }
+derive_more = "0.99"
+dotenv = "0.15"
+futures = "0.3"
+hex = "0.4"
+serde = "1.0"
+serde_json = "1.0"
+sqlx = { version = "0.7", features = ["runtime-tokio", "postgres", "chrono"] }
```

Comment:
Reading random posts on the internet warns me that `tokio_postgres` is "faster" than `sqlx`.. We also use `tokio_postgres` everywhere else AFAIK. I'm not opposed to `sqlx` as such, I'm just wondering if it's the most optimal library to achieve the goal. Is this something you've considere?

Reply:
Yes, the reason for using `sqlx` is the compile-time checked queries, since it saves us from having test cases catching bugs in the queries. It does come with a cost, especially compile time is up due to this, so it might not be worth it.
I have seen claims that `tokio_postgres` is faster and even 2x faster, but benchmarks can be misleading, so it is something we should compare ourselves. Right now the main concern is to reduce dev time.

Reply:
I've personally used sqlx in the past and I've found it to be very nice. Never had any performance issues with it, even for quite heavy services, but that's just anecdotal obviously.

But the _interface_ of the crate is very nice and much simpler than tokio_postgres.

Reply:
I imagine that most of the response time is spent by the database processing the query, so hopefully it does not matter too much and the compile time checked queries are speeding up development by removing the need for testing (to some degree at least). I don't expect a massive improvement by replacing .NET by Rust for the same reason.
We need to run with actual data to know for sure.

Reply:
Closing this for now, but feel free to take it up again